build/web-client/getting-started.md

```diff
@@ -0,0 +1,43 @@
+---
+next: ./installation/
+---
+
+# What's Nimiq?
+
+Nimiq is an open source crypto designed for simplicity and ease of use. With our browser-first approach, Nimiq works directly within web browsers without the need for additional software, allowing developers easy integration and users seamless interaction.
```

Comment:
I'd rephrase this as: providing easy integration for developers and seamless interaction for users.